README.md

```diff
@@ -45,115 +89,169 @@ environment variable `EXTRA_PKGS`. See [Environment Variables](#Environment Vari
 - systemd-bootchart
 - iptables-bin
 
-#### Clear Linux based packages security limitations  ####
+##### Clear Linux based packages security limitations
+
+Although the Clear Linux rootfs is constructed from `rpm` packages, Clear
+Linux itself is not an `rpm`-based Linux distribution (the software installed
+on a Clear Linux system is not managed using `rpm`).
 
-Clear Linux is not an rpm-based Linux distribution and
-the rpm packages are not signed, so there is no way
-to ensure that downloaded packages are trustworthy.
+The `rpm` packages used to generate the rootfs are not signed, so there is no
+way to ensure that downloaded packages are trustworthy.
 
-If you are willing to use Clear Linux based images, we encourage you
-to use the Clear Containers images provided from its website
-https://download.clearlinux.org/current/.
+If you are willing to use Clear Linux based images, official Clear Containers
+rootfs images can be obtained from https://download.clearlinux.org/releases.
 
-## Build guest kernel ##
+## Build guest kernel
 
-Clear Containers uses the Linux kernel, you can build a
-kernel compatible with Clear Containers using the make
-`kernel` target. This will clone the [Clear Container Kernel]
-if the directory `workdir/linux` does not exist, then will build it.
-On success the new kernel will be located in
-workdir/vmlinux.container
+Clear Containers uses the [Linux* kernel](https://www.kernel.org).
+
+To build a kernel compatible with Clear Containers using the make `kernel` target. This
+will clone the [Clear Container Kernel] in the `workdir/linux` directory
+(which will be created if necessary). On success two new kernel images will be created:
```

Comment:
Also, do we copy exactly what you put in https://github.com/clearcontainers/runtime/pull/597 into this section I wonder?

/me wishes we could import markdown files transparently into other markdown files to avoid duplication ;(